llvm/test/CodeGen/AMDGPU/verifier-sdwa-selection.mir

```diff
@@ -0,0 +1,22 @@
+# RUN: not --crash llc -mtriple=amdgcn -mcpu=gfx1030 --verify-machineinstrs -o - %s 2>&1 | FileCheck %s
```

Comment:
Don't need -verify-machineinstrs, can use -run-pass=none, and this belongs in test/MachineVerifier/AMDGPU

Reply:
I have moved and adjusted the test accordingly.